src/Sulu/Bundle/CustomUrlBundle/Request/CustomUrlRequestProcessor.php

```diff
@@ -27,36 +27,12 @@
  */
 class CustomUrlRequestProcessor implements RequestProcessorInterface
 {
-    /**
-     * @var CustomUrlManagerInterface
-     */
-    private $customUrlManager;
-
-    /**
-     * @var GeneratorInterface
-     */
-    private $generator;
-
-    /**
-     * @var WebspaceManagerInterface
-     */
-    private $webspaceManager;
-
-    /**
-     * @var string
-     */
-    private $environment;
-
     public function __construct(
-        CustomUrlManagerInterface $customUrlManager,
-        GeneratorInterface $generator,
-        WebspaceManagerInterface $webspaceManager,
-        $environment
+        private CustomUrlManagerInterface $customUrlManager,
+        private GeneratorInterface $generator,
+        private WebspaceManagerInterface $webspaceManager,
+        private ?string $environment
```

Comment:
We can add a type here because it would crash when using it to look up portal information anyways so better handle the type error early.